tests/tools/soft_sort_test.py

```diff
@@ -86,6 +86,51 @@ def test_sort_batch(self, rng: jax.random.PRNGKeyArray, topk: int):
     np.testing.assert_array_equal(xs.shape, expected_shape)
     np.testing.assert_array_equal(jnp.diff(xs, axis=axis) >= 0.0, True)
 
+  def test_multiv_cdf_quantiles(self):
+    n, d = 512, 3
+    keys = jax.random.split(jax.random.PRNGKey(0), 3)
```

Comment:
Please use the `rng` fixture as `def test_multiv_cdf_quantiles(self, rng: jax.random.PRNGKeyArray):`